src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_params.py

```diff
@@ -586,29 +587,77 @@ def register_blob_source_uri_arguments(scope):
-
-register_cli_argument('storage metrics show', 'services', help='The storage services from which to retrieve metrics info: (b)lob (f)ile (q)ueue (t)able. Can be combined.')
-register_cli_argument('storage metrics show', 'interval', help='Filter the set of metrics to retrieve by time interval.', **enum_choice_list(['hour', 'minute', 'both']))
-
-register_cli_argument('storage metrics update', 'services', help='The storage service(s) for which to update metrics info: (b)lob (f)ile (q)ueue (t)able. Can be combined.')
-register_cli_argument('storage metrics update', 'hour', help='Update the hourly metrics.', validator=process_metric_update_namespace, **enum_choice_list(['true', 'false']))
-register_cli_argument('storage metrics update', 'minute', help='Update the by-minute metrics.', **enum_choice_list(['true', 'false']))
-register_cli_argument('storage metrics update', 'api', help='Specify whether to include API in metrics. Applies to both hour and minute metrics if both are specified. Must be specified if hour or minute metrics are enabled and being updated.', **enum_choice_list(['true', 'false']))
-register_cli_argument('storage metrics update', 'retention', type=int, help='Number of days for which to retain metrics. 0 to disable. Applies to both hour and minute metrics if both are specified.')
-
-register_cli_argument('storage cors', 'max_age', type=int, help='The number of seconds the client/browser should cache a preflight response.', default="0")
-register_cli_argument('storage cors', 'origins', nargs='+', help='List of origin domains that will be allowed via CORS, or "*" to allow all domains.')
-register_cli_argument('storage cors', 'methods', nargs='+', help='List of HTTP methods allowed to be executed by the origin.', **enum_choice_list(['DELETE', 'GET', 'HEAD', 'MERGE', 'POST', 'OPTIONS', 'PUT']))
-register_cli_argument('storage cors', 'allowed_headers', nargs='+', help='List of response headers allowed to be part of the cross-origin request.')
-register_cli_argument('storage cors', 'exposed_headers', nargs='+', help='List of response headers to expose to CORS clients.')
-
-register_cli_argument('storage cors add', 'services', help='The storage service(s) for which to add the CORS rule: (b)lob (f)ile (q)ueue (t)able. Can be combined.')
-
-register_cli_argument('storage cors clear', 'services', help='The storage service(s) for which to clear CORS rules: (b)lob (f)ile (q)ueue (t)able. Can be combined.')
-
-register_cli_argument('storage cors list', 'services', help='The storage service(s) for which to list the CORS rules: (b)lob (f)ile (q)ueue (t)able. Can be combined.')
+#######################################
+# storage cors commands parameters
+#######################################
+
+with CommandContext('storage cors') as c:
+    c.reg_arg('max_age', type=int, default='0',
+              help='The number of seconds the client/browser should cache a preflight response.')
```

Comment:
Should we move this help text into the _help.py file instead of here, especially given how specialized the CORS commands are.